examples/run_generic_inference_engine.py

```diff
@@ -2,19 +2,34 @@
 from unitxt.inference import GenericInferenceEngine
 
 if __name__ == "__main__":
```

Comment:
if its an example you need to explain in text what you are doing here

Reply:
added